fastly.toml

```diff
@@ -3,3 +3,15 @@ name = "Default starter for Rust"
 description = "Get to know the Fastly Compute@Edge environment with a basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
 authors = ["<oss@fastly.com>"]
 language = "rust"
+
+[setup]
+  [[setup.backends]]
+    name = "backend_name"
+    prompt = "Origin server able to serve /solutions/ path"
+    address = "developer.fastly.com"
+    port = 443
+  [[setup.backends]]
+    name = "other_backend_name"
+    prompt = "Origin server able to serve /anything/ path (and change of Host header to example.com)"
```

Comment:
I appreciate this prompt is overly verbose but as this 'starter kit' is changing the Host header, I feel we should be _explicit_ about it otherwise the user might not notice that line of code and deploying the service might break if they also set their own backend to one that fails if the host doesn't match.